decoder/vaapidecoder_h264.cpp

```diff
@@ -1370,11 +1370,6 @@ bool VaapiDecoderH264::fillPicture(const PicturePtr& picture,
     picParam->seq_fields.bits.delta_pic_order_always_zero_flag
         = sps->delta_pic_order_always_zero_flag;
 
-    picParam->num_slice_groups_minus1 = pps->num_slice_groups_minus1;
```

Comment:
will not break conformance, these fields are not used. and As libva2.0 have marked them as deprecated, we need to remove them to avoid compile errors.